spec/requests/healthcheck_search_spec.rb

```diff
@@ -0,0 +1,19 @@
+RSpec.describe "/healthcheck/search" do
+  it "returns distinct organisations ordered by title" do
+    get "/healthcheck/search"
+    json = JSON.parse(response.body)
+
+    expect(json["checks"]).to include("aggregations")
+      .and(include("search_last_month"))
+      .and(include("search_last_six_months"))
+      .and(include("search_last_thirty_days"))
+      .and(include("search_last_three_months"))
+      .and(include("search_last_twelve_months"))
+  end
+
+  it "is not cacheable" do
+    get "/healthcheck"
```

Comment:
```suggestion
    get "/healthcheck/search"
```

And similar to my comment above about a `before` block.